e2e/e2etests/test_sui_withdraw_and_call_revert_with_call.go

```diff
@@ -25,20 +23,9 @@ func TestSuiWithdrawAndCallRevertWithCall(r *runner.E2ERunner, args []string) {
 	targetPackageID := r.SuiExample.PackageID.String()
 	amount := utils.ParseBigInt(r, args[0])
 
-	// Given example contract on_call function arguments
-	argumentTypes := []string{
-		r.SuiExample.TokenType.String(),
-	}
-	objects := []string{
-		r.SuiExample.GlobalConfigID.String(),
-		r.SuiExample.PoolID.String(),
-		r.SuiExample.PartnerID.String(),
-		r.SuiExample.ClockID.String(),
-	}
-
-	// define an invalid address to cause 'on_call' failure
+	// create the payload for 'on_call' with invalid address
 	invalidAddress := "8f569597ebca884b784d32678a6f"
```

Comment:
Nit : Consider moving this address to the sample package

Reply:
I was thinking that the sample generator should still generate a valid address. So I simply define a constant for this revert test.

Reply:
I mean a function like InvalidSuiAddress, and just return the constant. 
I see its already used in two places and we might use it in more in the future

Reply:
IC. Now I just realized that using a truncated sui address also works: https://github.com/zeta-chain/node/pull/3831/commits/1c067b999a2e51d58384f0b3227ef73daea2e868